plugins/etcd/src/main/java/org/radargun/service/EtcdService.java

```diff
@@ -0,0 +1,79 @@
+package org.radargun.service;
+
+import java.io.IOException;
+
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.Lifecycle;
+import org.radargun.traits.ProvidesTrait;
+import org.radargun.utils.Utils;
+
+/**
+ *
+ * Usage instructions:
+ * You need to compile etcd on your target test system, and point to its directory with the distributionDir property.
+ * Alternatively you can zip the compiled distribution, point to the zip with the distributionZip property and RG will unzip it to distributionDir for you.
+ *
+ * Note that etcd saves some data in the distribution directory, and you need to delete it between runs. Using the distributionZip
+ * property is preferred, so the distribution is 'clean' with each run.
+ *
+ */
+@Service(doc = "Etcd")
+public class EtcdService implements Lifecycle {
+
+   private Process server;
+
+   @Property(doc = "Directory of the etcd distribution.", optional = false)
+   protected String distributionDir;
+
+   @Property(doc = "Distribution zip. If set, the zip will be extracted to the 'distributionDir' directory.")
+   protected String distributionZip;
+
+   @ProvidesTrait
+   public EtcdService getSelf() {
+      return this;
+   }
+
+   @Override
+   public void start() {
+
+      try {
+         if (distributionZip != null) {
+            Utils.unzip(distributionZip, distributionDir);
+            // the extraction erases the executable bits
+            Utils.setPermissions(distributionDir + "/etcd", "rwxr-xr-x");
+         }
+      } catch (IOException e) {
+         throw new RuntimeException("Failed to prepare etcd distribution!", e);
+      }
+
+      ProcessBuilder processBuilder = new ProcessBuilder(distributionDir + "/etcd");
+      processBuilder.inheritIO();
+
+      try {
+         server = processBuilder.start();
+      } catch (IOException e) {
+         throw new RuntimeException("Cannot start the etcd server.", e);
+      }
+
+   }
+
+   /**
+    * Stop the member etcd process
+    *
+    * https://coreos.com/etcd/docs/latest/v2/admin_guide.html
+    * ssh 10.0.1.11
+    * kill `pgrep etcd`
+    */
+   @Override
+   public void stop() {
+
```

Comment:
I can see that you're using this blank line in some methods but not all. Probably good to unify the approach.